### ISSUE TITLE
Remapping landmark rules to proper WCAG SC (4Q25)

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#2242]
**Describe the bug**
Based on the team discussion, the WCAG mapping of the landmark rules should be:
1.  The aria_content_in_landmark rule to 1.3.1
2.  All other landmark rules to 1.3.1

The help content for each rule needs to be updated accordingly.

**Comment by tombrunet** [comment]
@shunguoy to schedule a 30 minute meeting to discuss the IBM requirement for landmarks and what we want to do with these rules overall.
